codespell_lib/data/dictionary.txt

```diff
@@ -7104,6 +7114,7 @@ depricates->deprecates
 depricating->deprecating
 dequed->dequeued
 derageable->dirigible
+derails->derails, details,
```

Comment:
For now can you remove this? See for example #1142

Reply:
OK, `derails->derails, details,` has been removed

Reply:
IMO, we should have a separate dictionary or perhaps flag that lets the user search these type of typos.